backend/Model/RoomModel.php

```diff
@@ -87,15 +89,16 @@ public function createRoom(
         } while ($roomUrlIsNotUnique);
 
         $this->title = $title;
-        $this->shouldJoin = $shouldJoin;
```

Comment:
Why is this one deleted?

Reply:
`shouldJoin` does not need to be saved on the `Room` table, it is already evident from the `RoomUser` table.